inc/api/apirest.class.php

```diff
@@ -338,9 +342,19 @@ private function getItemtype($index = 0, $recursive = true, $all_assets = false)
                $itemtype                            = $additional_itemtype;
             }
 
```

Comment:
Fix a bug introduced by #6850, the special case "AllAssets" is not a real class and can't be instantiated by reflection.

Reply:
Please open another PR with fixes, this one must be applied on 9.4/bf

Reply:
Done in #7080